spec/std/string_spec.cr

```diff
@@ -653,7 +653,8 @@ describe "String" do
 
       describe "with offset" do
         assert { "foo baro baz".rindex("o b", 6).should eq(2) }
-        assert { "foo baro baz".rindex("fg").should be_nil }
```

Comment:
There is this in `"with offset"` section, but it has no offset option and it exists already [before](https://github.com/MakeNowJust/crystal/blob/e4603171caa96024ed2eb40c0b37eeeab0949c79/spec/std/string_spec.cr#L651) itself, so I thought it is just copy-and-paste mistaking.